docs/main_concepts.md

```diff
@@ -117,7 +119,7 @@ some elements in the interactive table.
    the Styler object. To install Jinja2, run: pip install jinja2.
```

Comment:
Is this an artifact of something? Why is pip install jinja2 part of the docs?

Reply:
This isn't a new addition to the docs. Historically, jinja2 was a dependency to use Pandas' Styler object in Streamlit. I will have to check whether this is still the case. If it is, I'll check if jinja2 is installed as a Streamlit dependency. If so, we can remove this note altogether.

To clarify, are you taking issue with the entire note or just `pip install jinja2` being a part of the note? If it's the latter,  we can remove it from the note right away.

Reply:
I can confirm that `jinja2` is automatically installed as a dependency during `pip install streamlit`. Will delete the note :+1: